src/Sentry/Sentry.csproj

```diff
@@ -93,35 +94,17 @@
       <_OSArchitecture>$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)</_OSArchitecture>
     </PropertyGroup>
     <ItemGroup>
-      <SentryCLIDownload
```

Comment:
Undo unrelated whitespace changes please.  (We really should figure out how to keep this from happening.)